frontend/src/ui/Input.tsx

```diff
@@ -287,3 +288,36 @@ export const CopyableInput: React.FC<CopyableInputProps> = ({
         </div>
     );
 };
+
+type DisplayOptionGroup<TFieldValues extends FieldValues> = {
+    form: UseFormReturn<TFieldValues>;
+    type: React.HTMLInputTypeAttribute;
```

Comment:
Interesting idea, but this would allow passing `type: "text"` right? And that would surely break this component? So maybe say `type: "radio" | "checkbox"`?